modules/install-sensor-roxctl.adoc

```diff
@@ -15,7 +15,7 @@ The following steps describe adding Sensor by using the RHACS portal.
 . Specify a name for the cluster.
 . Provide appropriate values for the fields based on where you are deploying the Sensor.
 ** If you are deploying Sensor in the same cluster, accept the default values for all the fields.
-** If you are deploying into a different cluster, replace `central.stackrox:443` with a LoadBalancer, NodePort, or other address, including the port number, that is accessible from the other cluster.
+** If you are deploying into a different cluster, replace `central.stackrox.svc:443` with a LoadBalancer, NodePort, or other address, including the port number, that is accessible from the other cluster.
```

Comment:
Just pointing out that `download-sensor-bundle.adoc` uses the preferred<sup>*</sup> "load balancer, node port, or...".

<sup>*</sup> Preferred by me and seems like a "general reference" according to the [API formatting guidelines](https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#api-object-formatting).